### ISSUE TITLE
Question: Why are the labels in the train and test.csv file single digit numbers ?

### ISSUE THREAD

**Issue by Aayush2003Gupta** [abedicodes/ResNet-TCN#8]
Thanks a lot for the repo!!
I was not able to understand the labels of the train and test.csv given  in this repository, the labels are given to be single digit numbers 0,1,2,3 but shouldn't it be a 4-tuple ( representing levels for each emotion as mentioned in the csv provided in the DAiSEE dataset itself)

I'm unable to find any test.py(for direct inference through webcam or on my own images), also I would be highly grateful if you could provide the pretrained weights of the model, 

Thanks for the help!!

**Comment by Aayush2003Gupta** [comment]
I am not sure but after reading the paper of DAiSEE dataset , I think that for each **Affective State** namely :-
1. Boredom 
2. Engaged 
3.  Confused 
4.  Frustrated 
We would have to make different model  . 1 for Boredom , 1 for Engaged , 1 for Confused and 1 for Frustrated. Not sure but maybe this is what I have understood as of now.

In Section 3.2 of the paper,
`Motivated by recent work in intelligent tutoring systems [54], our dataset consists of labels for `**four affective states** `related to user engagement, viz., `**engagement,
frustration, confusion, and boredom**`

`_Each affective state is benchmarked individually using the given models:_` This line in Section 4.2 of the DAiSEE paper made me think this way

**Comment by Aayush2003Gupta** [comment]
The model is `only made for the Engagement level prediction` , it is not meant for Affective states of Boredom , Confusion and Frustration therefore it only asks for the labels of the Engagement levels in the label column in the train and test csv
